core/deployment/src/main/java/io/quarkus/deployment/logging/LoggingResourceProcessor.java

```diff
@@ -103,14 +100,6 @@ void setupLoggingStaticInit(LoggingSetupTemplate setupTemplate) {
         setupTemplate.initializeLoggingForImageBuild();
     }
 
-    @BuildStep
-    ConfigurationCustomConverterBuildItem setUpLevelConverter() {
-        return new ConfigurationCustomConverterBuildItem(
-                200,
-                Level.class,
-                LevelConverter.class);
```

Comment:
This was removed because this converter is now included in services file and therefore there is no need to produce `ConfigurationCustomConverterBuildItem`.